### PR TITLE
all: Add grpc-xds

### PR DESCRIPTION
**Pull request by ejona86**
This will fix reporting of xDS code coverage.

CC @voidzcy

### CODE REVIEW

File: all/build.gradle [hunk 1]
```diff
@@ -18,6 +18,7 @@ def subprojects = [
     project(':grpc-protobuf-lite'),
     project(':grpc-stub'),
     project(':grpc-testing'),
+    project(':grpc-xds'),
```
Comment (dapengzhang0) [inline all/build.gradle:21]: Is this collection used for both javadoc and jacoco?

Comment (ejona86) [reply all/build.gradle:21]: Yes. It also adds a dependency in the pom to grpc-all.

**Comment by ejona86** [comment]
Those are part of #2494, which still needs some love fixing up the jacoco report. (xds was a bit easier since generated code was in non-grpc packages.)

**Comment by ejona86** [comment]
Err.. I guess I hadn't planned to for grpc-testing-proto, but you're right, we should for the javadoc.

**Pull request merged**